src/pages/my_page/user_profile/MyFieldAndBackground.tsx

```diff
@@ -51,18 +51,37 @@ export const MyFieldAndBackground = () => {
       no: personBackground.no ?? '',
       en: personBackground.en ?? '',
     },
-    keywords: personKeywords ?? [],
+    keywords: personKeywords
+      ? personKeywords.map((keyword: any) => {
```

Comment:
`personKeywords` burde allerede ha en type her, som vi ikke burde trenge å overstyre, eller misser jeg noe?
```suggestion
      ? personKeywords.map((keyword) => {
```